src/olympia/addons/migrations/0010_auto_20200604_1319.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.12 on 2020-06-04 13:19
```

Comment:
Nit: Don't we tend to rename the migration files to describe what they are doing?

Reply:
I rarely do this.